pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
this change should be reverted as it will probably fail the boilerplate (license header) verification.

Reply:
fixed thanks